.github/workflows/push-lacchain-environment.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
           REACT_APP_HASURA_URL: 'https://graphql-lacchain.eosio.online/v1/graphql'
           REACT_APP_EOS_API_NETWORK_NAME: 'lacchain'
           REACT_APP_EOS_API_NETWORK_LABEL: 'LACChain EOSIO'
-          REACT_APP_EOS_API_NETWORK_LOGO: 'https://latamlink.io/images/latamlink_logo-header.svg'
+          REACT_APP_EOS_API_NETWORK_LOGO: 'https://lacchain.eosio.online/lacchain.png'
```

Comment:
this should be the color version of the logo

Reply:
![image](https://user-images.githubusercontent.com/5632966/120103921-2f781080-c10f-11eb-986d-9b800ab3de28.png)